src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -229,7 +230,7 @@ trait BeamHelper extends LazyLogging {
           bind(classOf[TollCalculator]).asEagerSingleton()
 
           // Override EventsManager
-          bind(classOf[EventsManager]).to(classOf[LoggingParallelEventsManager]).asEagerSingleton()
+          bind(classOf[EventsManager]).to(classOf[EventsManagerImpl]).asEagerSingleton()
```

Comment:
`LoggingParallelEventsManager` is somehow replaced by `EventsManagerImpl`

Reply:
done